import sys 
sys.path.append(".")



from utils.exception_ import CustomException 


print(CustomException)

